core/model/src/main/java/org/activityinfo/model/type/number/Quantity.java

```diff
@@ -77,6 +83,7 @@ public int hashCode() {
         return ((int) (temp ^ (temp >>> 32))) ^ (units != null ? units.hashCode() : 0);
     }
 
+    @Deprecated
     public boolean hasUnits() {
```

Comment:
Were there problems removing the units field entirely?

Reply:
I just tagged the unit related aspects of Quantity as deprecated. I can update the pull request and actually remove all these methods/fields from the model if preferred?

Reply:
Give it a try. if all the tests pass than scrap 'em, otherwise, we can tackle it later.

Reply:
Alright!